docs/rfcs/0010-custom-stage.md

```diff
@@ -0,0 +1,210 @@
+- Start Date: 2023-02-20
+- Target Version: 0.41.4
+
+# Summary
+
+This RFC introduces a new way to enable users to use “custom stages” that users defined in their pipelines.
+
+# Motivation
+
+Currently, users can use only stages that PipeCD have already defined. But some users want to define new stages by their use-cases as bellow. 
+
+- Deploying infrastructure by tools other than that PipeCD supports (terraform and kubernetes) such as SAM, cloud formation….
+- Running End to End tests
```

Comment:
should not be listed here (CI roles)

Reply:
I think this can be included CD roles because there is some cases where developers want to check pods run.